Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/interpolated_corrected_curvatures.h

```diff
@@ -0,0 +1,1553 @@
+*                    as key type and `%Point_3` as value type}
+*     \cgalParamDefault{`boost::get(CGAL::vertex_point, pmesh)`}
+*     \cgalParamExtra{If this parameter is omitted, an internal property map for
+*                     `CGAL::vertex_point_t` must be available in `PolygonMesh`.}
+*   \cgalParamNEnd
+*
+*   \cgalParamNBegin{vertex_normal_map}
+*     \cgalParamDescription{a property map associating normal vectors to the vertices of `pmesh`}
+*     \cgalParamType{a class model of `ReadablePropertyMap` with
+*                    `boost::graph_traits<PolygonMesh>::%Vertex_descriptor`
+*                    as key type and `%Vector_3` as value type}
+*     \cgalParamDefault{`get(dynamic_vertex_property_t<GT::Vector_3>(), pmesh)`}
+*     \cgalParamExtra{If this parameter is omitted, vertex normals will be
+*                     computed using compute_vertex_normals()}
+*   \cgalParamNEnd
+*
+*   \cgalParamNBegin{ball_radius}
+*     \cgalParamDescription{a scalar value specifying the radius used for expanding curvature measures
+*                           by summing measures of faces inside a ball of this radius centered at the
+*                           vertex expanded from. The summed face measures are weighted by their
+*                           inclusion ratio inside this ball}
+*     \cgalParamType{`GT::FT`}
+*     \cgalParamDefault{`-1`}
+*     \cgalParamExtra{If this parameter is omitted (`-1`), the expansion will just by a weightless sum of
+*                     measures on faces around the vertex}
+*   \cgalParamNEnd
+*
+* \cgalNamedParamsEnd
+*
+* @see `interpolated_corrected_mean_curvature()`
+* @see `interpolated_corrected_gaussian_curvature()`
+* @see `interpolated_corrected_curvatures()`
+*/
+template<typename PolygonMesh, typename VertexCurvatureMap,
+  typename NamedParameters = parameters::Default_named_parameters>
+  void interpolated_corrected_principal_curvatures_and_directions(const PolygonMesh& pmesh,
+    VertexCurvatureMap& vcm,
+    const NamedParameters& np = parameters::default_values())
+{
+  interpolated_corrected_curvatures(pmesh, np.vertex_principal_curvatures_and_directions_map(vcm));
+}
+
+
+/**
+  * \ingroup PMP_corrected_curvatures_grp
+  * Computes the interpolated corrected curvatures at a certain vertex, based on the provided pointers.
+  * By providing mean, gaussian and/or principal curvature pointers as named parameters, the user
+  * can choose which curvatures to compute.
+  * The pointers are used to store the computed curvatures.
+  * The user is responsible for the memory management of the pointers.
+  *
+  * @tparam PolygonMesh a model of `FaceListGraph`
+  * @tparam NamedParameters a sequence of \ref pmp_namedparameters "Named Parameters"
+  *
+  * @param pmesh the polygon mesh
+  * @param v the vertex of `pmesh` to compute the curvatures at
+  * @param np optional sequence of \ref pmp_namedparameters "Named Parameters" among the ones listed below
```

Comment:
In the overnight documentation build https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.6-Ic-168/logs_master/Polygon_mesh_processing.log we see:
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-168/include/CGAL/Polygon_mesh_processing/interpolated_corrected_curvatures.h:1278: warning: unable to resolve reference to 'pmp_namedparameters' for \ref command
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-168/include/CGAL/Polygon_mesh_processing/interpolated_corrected_curvatures.h:1282: warning: unable to resolve reference to 'pmp_namedparameters' for \ref command
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-168/include/CGAL/Polygon_mesh_processing/interpolated_corrected_curvatures.h:1425: warning: unable to resolve reference to 'pmp_namedparameters' for \ref command
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-168/include/CGAL/Polygon_mesh_processing/interpolated_corrected_curvatures.h:1429: warning: unable to resolve reference to 'pmp_namedparameters' for \ref command
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-168/include/CGAL/Polygon_mesh_processing/interpolated_corrected_curvatures.h:1359: warning: unable to resolve reference to 'pmp_namedparameters' for \ref command
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-168/include/CGAL/Polygon_mesh_processing/interpolated_corrected_curvatures.h:1363: warning: unable to resolve reference to 'pmp_namedparameters' for \ref command
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-168/include/CGAL/Polygon_mesh_processing/interpolated_corrected_curvatures.h:1491: warning: unable to resolve reference to 'pmp_namedparameters' for \ref command
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-168/include/CGAL/Polygon_mesh_processing/interpolated_corrected_curvatures.h:1495: warning: unable to resolve reference to 'pmp_namedparameters' for \ref command
```

might it be that the reference has to be to `bgl_namedparameters` ?